src/types.ts

```diff
@@ -18,6 +18,18 @@ export type LnUrlPayResponse = {
   commentAllowed?: number;
   rawData: { [key: string]: string | number };
   allowsNostr: boolean;
+  payerData?: Partial<LUD18ServicePayerData>;
+}
+
+export type LUD18ServicePayerData = {
```

Comment:
good example where I dislike TS: payerdata can be anything. the TS here does not include the content uri for example.
how can we keep this open?

Reply:
` & Record<string, unknown>` seems to be ok. It's open but still suggests the typed properties